packages/test-runner/src/fixtures.ts

```diff
@@ -274,3 +279,16 @@ export function rerunRegistrations(file: string, scope: Scope) {
   for (const registration of lookupRegistrations(file, scope).values())
     registrations.set(registration.name, registration);
 }
+
+export async function generateTestAssetName(info: TestInfo, suffix: string): Promise<string> {
```

Comment:
If we are making it a part of the framework, we can expose this function on the Test

Reply:
Actually we can use a fixture for that.

Reply:
We could, that'd actually be nice because user will be able to redefine it.